stl/inc/format

```diff
@@ -2229,6 +2229,27 @@ public:
     }
 };
 
+#if _HAS_CXX23
+template <class _CharT>
+struct _Phony_fmt_iter_for {
+    using iterator_category = output_iterator_tag;

```

Comment:
We should probably remove all of these nested types (we don't want this iterator archetype to have anything not required so as to avoid users writing `formatter` specializations that depend on non-portable behavior).

Reply:
Ditto followup.